### DISCUSSION TITLE
Elrs FlySky FS-i6

### DISCUSSION THREAD

**Discussion by FOd2223** [ExpressLRS/ExpressLRS#2509] [General]
Hello! I encountered the same problem. I have a bait delivery boat, it runs on Flysky fs-i6 equipment, frequency 2.4 GHz.  On the boat, the echo sounder also operates at 2.4 GHz. There is a problem with a frequency conflict. Tell me how to make the flysky fs-i6 equipment using elrs work at a different frequency?

**Comment by sk8board** [comment]
ELRS and FlySky use different protocols. There is not a way to use your FlySky FS-i6 with ELRS. 

While 2.4 GHz ELRS will likely be more resilient to 2.4 GHz interference than FlySky protocol, 2.4 GHz ELRS will still struggle with 2.4 GHz interference. If you cannot resolve the 2.4 GHz noise problem from your echo sounder, then you should consider 900 MHz ELRS.

https://www.radiomasterrc.com/collections/bandit
